httpserver/cache.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package httpserver
+
+import (
+	"time"
+
+	"github.com/deislabs/ratify/pkg/executor/types"
+	utilcache "k8s.io/apimachinery/pkg/util/cache"
```

Comment:
There was a question in the community call on why we need to use api machinery cache and not ristretto?